PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,7 @@
 -->
 
 ## Purpose
+
```

Comment:
Please rollback these changes and leave only the line which you added the step to check logging. Thank you